src/style.css

```diff
@@ -1 +1,60 @@
-/* Styles go here */
+body {
+  margin: 0;
+  background-color: #e5e5e5;
+}
+
+.nav {
+  margin-right: 50px;
+}
+
+header {
+  display: flex;
+  background-color: #fff;
+  justify-content: space-between;
+  align-items: center;
+  flex-wrap: nowrap;
+}
+
+.logo {
+  margin: 10px 0 10px 50px;
+  height: 40px;
+  width: 40px;
+}
+
+.text {
+  height: 60px;
+  text-align: center;
```

Comment:
`inline-block` make links width to be minimal, according to the width of content, you don't need this rule
```suggestion
```